sdk/spring/azure-spring-data-cosmos/src/main/java/com/azure/spring/data/cosmos/repository/CosmosRepository.java

```diff
@@ -15,7 +16,7 @@
  * pagination and sorting abstraction.
  */
 @NoRepositoryBean
-public interface CosmosRepository<T, ID extends Serializable> extends PagingAndSortingRepository<T, ID> {
+public interface CosmosRepository<T, ID extends Serializable> extends CrudRepository<T, ID>, PagingAndSortingRepository<T, ID> {
```

Comment:
Please revert - not necessary PagingAndSortedRepository already extends CrudRepository - so CosmosRepository is implicitly extending CrudRepository already